src/zjs_aio.c

```diff
@@ -19,7 +19,7 @@ static struct nano_sem aio_sem;
 
 typedef struct aio_handle {
     jerry_value_t pin_obj;
-    int32_t callback_id;
+    int16_t callback_id;
```

Comment:
This doesn't have anything to do with jerryscript. The callback ID used to be stored as a int32_t, but there is not a need for that many callbacks, so we are re-evaluating and changing it to a int16_t to save 2 bytes.